code/__HELPERS/lighting.dm

```diff
@@ -4,6 +4,11 @@
 	// Consider removing it someday? (I wonder if we made emissives blend right we could make alpha actually matter. dreams man, dreams)
 	var/mutable_appearance/appearance = mutable_appearance(icon, icon_state, layer, offset_spokesman, EMISSIVE_PLANE, 255, appearance_flags | EMISSIVE_APPEARANCE_FLAGS, offset_const)
 	appearance.color = GLOB.emissive_color
+
+	if(icon_state && !icon_exists(icon, icon_state, FALSE))
```

Comment:
1. You need to put this behind testing flags. Take a look at the `focus_only` and `PERFORM_ALL_TESTS` idiom. This proc is *extremely* expensive and this will hurt init times a lot.
2. Named parameter for `FALSE`.

Reply:
Think I got the testing flags done correctly, thanks for catching that! Hope I did it right

Added the named param as well 👍